input/fsh/operations/moped-operation-entlassen.fsh

```diff
@@ -25,33 +31,41 @@ Usage: #definition
 * parameter[+]
   * name = #zeitpunkt
   * use = #in
-  * min = 1
+  * min = 0
   * max = "1"
-  * documentation = "Der *zeitpunkt* Parameter definiert zu welchem Zeitpunkt die Beurlaubung startet."
+  * documentation = "Der *zeitpunkt* Parameter definiert zu welchem Zeitpunkt die Entlassung stattfindet. Verpflichtend bei stationären Entlassungen."
```

Comment:
Entlassungen können nur stationär sein. Müssen wir die Operation dann umbenennen, wenn es ein ambulanter Fall ist?

Reply:
frage ausgelagert in https://github.com/HL7Austria/ELGA-MOPED-R5/issues/497#issuecomment-3026621816